packages/modelviewer.dev/examples/scenegraph/index.html

```diff
@@ -565,6 +565,48 @@ <h2 class="demo-title">Exporter with options</h2>
       </div>
     </div>
 
+    <div class="sample">
+      <div id="pickMaterialExample" class="demo"></div>
+      <div class="content">
+        <div class="wrapper">
+          <div class="heading">
+            <h2 class="demo-title">Swap textures - Normals/Occlusion/Emission</h2>
+            <p>As above, you can change these values in AR, but only in WebXR
+              mode. iOS Quick Look reflects these texture changes so long as the
+              USDZ is auto-generated.</p>
+          </div>
+          <example-snippet stamp-to="pickMaterialExample" highlight-as="html">
+            <template>
+<model-viewer id="pickMaterial" camera-controls src="../../shared-assets/models/glTF-Sample-Models/2.0/Duck/glTF-Binary/Duck.glb" ar ar-modes="webxr scene-viewer quick-look" alt="A Material Picking Example">
```

Comment:
Does the duck have more than one material? We should probably pick a model that does.

Reply:
Incoming...